cypress/e2e/signIn.cy.js

```diff
@@ -2,6 +2,24 @@
 
 describe('Sign In page', () => {
   it('should provide an ability to log in', () => {
+    cy.visit('https://conduit.mate.academy/user/login');
 
+    cy.get('h1')
+      .should('contain.text', 'Sign in');
+
+    // email entering
+    cy.get(':nth-child(1) > .form-control')
+      .type('bybeg@mailinator.com');
+
+    // password entering
+    cy.get(':nth-child(2) > .form-control')
+      .type('Pa$$w0rd!');
+
+    cy.get('.btn')
+      .click();
+
+    cy.get(':nth-child(4) > .nav-link')
+      .contains('estetvelieaoff')
+      .should('be.visible');
```

Comment:
Avoid using CSS selectors that rely on the structure of the HTML. Use more stable selectors instead. Also, it's better to assert on user-visible content, like the text of the link, rather than asserting that an element is visible.